pendulum_control/include/pendulum_control/rtt_executor.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef PENDULUM_CONTROL__RTT_EXECUTOR_HPP_
 #define PENDULUM_CONTROL__RTT_EXECUTOR_HPP_
 
+#include <string.h>
```

Comment:
Since this is a C++ header, I think `#include <cstring>` would be more appropriate.